src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/DebugPropertyPage/RemoveStartArgumentsActionHandler.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.Xml;
+using Microsoft.VisualStudio.ProjectSystem.VS.PropertyPages.Designer;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Properties;
+
+[Export(typeof(ILinkActionHandler))]
+[ExportMetadata("CommandName", "RemoveStartArguments")]
+internal class RemoveStartArgumentsActionHandler : ILinkActionHandler
+{
+    private const string PropertyNameToDelete = "StartArguments";
+    private const string UserSuffix = ".user";
+    private string? _xmlFile;
+
+    public Task HandleAsync(UnconfiguredProject project, IReadOnlyDictionary<string, string> editorMetadata)
+    {
+        _xmlFile = project.FullPath + UserSuffix;
+
+        _ = project.Services.ThreadingPolicy.JoinableTaskFactory.RunAsync(RemoveElementAsync);
+
+        return Task.CompletedTask;
+    }
+
+    private Task RemoveElementAsync()
+    {
+        XmlDocument doc = new XmlDocument() { XmlResolver = null };
+
+#pragma warning disable CA3075 // Insecure DTD processing in XML
+        doc.Load(_xmlFile);
```

Comment:
File IO and parsing can throw exceptions. For example, if the file is missing (you can never assume it's present until you've successfully opened it), locked by some other process, or if it contains invalid XML.

You should handle exceptions here. Ditto below on `Save`.

Reply:
exception handled